rhcos.tdl

```diff
@@ -0,0 +1,13 @@
+<template>
+    <name>rhcos</name>
+    <os>
+        <name>RHEL-7</name>
+        <version>0</version>
+        <arch>x86_64</arch>
+        <install type='url'>
+            <url>https://dl.fedoraproject.org/pub/fedora/linux/releases/28/Everything/x86_64/os/</url>
```

Comment:
Minor: I am guessing this url will be swapped, once we publish the composed ostree repo(created via the other pipeline) to a similar url?

Reply:
Ah, NVM, I think I see, I was confused for a while, I am guessing 
`sh "sed -i 's,\\(<url>\\).*\\(<\\/url\\),\\1${env.INSTALLER_TREE_URL}\\2,' rhcos.tdl"` does the url swap alrdy?

Reply:
Yes, that's correct.